### PR TITLE
wildcard object and doctests

### PR DESCRIPTION
**Pull request by llllllllll**

### CODE REVIEW

File: zipline/pipeline/factors/factor.py [hunk 1]
```diff
@@ -333,9 +333,9 @@ class Factor(RestrictedDTypeMixin, ComputableTerm):
     Factors.  For example, constructing a Factor that computes the average of
     two other Factors is simply::
 
-        >>> f1 = SomeFactor(...)
-        >>> f2 = SomeOtherFactor(...)
-        >>> average = (f1 + f2) / 2.0
+        >>> f1 = SomeFactor(...)  # doctest: +SKIP
```
Comment (ssanderson) [inline zipline/pipeline/factors/factor.py]: Is there not a way to tell doctest to skip an entire docstring?

Comment (ssanderson) [inline zipline/pipeline/factors/factor.py]: (I think these will show up in autogenerated docs unless sphinx is smart about these.)

Comment (llllllllll) [inline zipline/pipeline/factors/factor.py]: It is smart about these (or has the ability to be). We do this in the blaze docs and they don't appear in the output.

**Comment by llllllllll** [comment]
> +2.3%

nice

File: zipline/utils/events.py [hunk 2]
```diff
@@ -301,8 +301,6 @@ class AfterOpen(StatelessRule):
     """
     A rule that triggers for some offset after the market opens.
     Example that triggers after 30 minutes of the market opening:
-
-    >>> AfterOpen(minutes=30)
```
Comment (ssanderson) [inline zipline/utils/events.py]: Why did we remove this?

Comment (ssanderson) [inline zipline/utils/events.py]: (If we do remove it, we should remove the sentence above referring to it, but it seems useful to have a usage example.)

Comment (llllllllll) [inline zipline/utils/events.py]: what is it showing us?

Comment (ssanderson) [inline zipline/utils/events.py]: (It looks like `minutes` is the wrong kwarg here though?)

Comment (llllllllll) [inline zipline/utils/events.py]: Okay, I will update the docstring

**Pull request merged**